android/src/main/java/com/testfairy/react/TestFairyModule.java

```diff
@@ -411,25 +411,37 @@ public void setFeedbackOptions(final ReadableMap options) {
 			@Override
 			public void run() {
 				FeedbackOptions.Builder feedbackOptions = new FeedbackOptions.Builder();
-				ReadableType defaultTextType = options.getType("defaultText");
-				if (defaultTextType == ReadableType.String) {
-					feedbackOptions.setDefaultText(options.getString("defaultText"));
+				if(options.hasKey("defaultText")) {
+					ReadableType defaultTextType = options.getType("defaultText");
+					if (defaultTextType == ReadableType.String) {
+						feedbackOptions.setDefaultText(options.getString("defaultText"));
+					}
```

Comment:
@eladgel Missing code block if hasKey() && defaultTextType != Readable.String :)
We'll add that. Thanks for the PR!!